AfrikaBurn/Model/PersistentStore.swift

```diff
@@ -37,7 +38,53 @@ class PersistentStore {
     
     func elements() -> Results<AfrikaBurnElement> {
         let realm = createRealm()
-        return realm.objects(AfrikaBurnElement.self)
+        
+        // filter out known test or dud IDs, or elements asked not be listed
+        let invalidElementIds = [
```

Comment:
I kept seeing test data in the table view  so I wanted to clean up the data a bit. 
I went through the CSV and found the ones that were obviously dummy data/test data and a couple that specifically asked to not be put in the list. I remember Wayne mentioning this on slack.
You can look up the same IDs in the CSV and see if you agree.

If you have a better approach than this let me know.